app/controllers/feeds_controller.rb

```diff
@@ -46,10 +53,15 @@ def update
 
   # DELETE /feeds/1
   def destroy
-    @feed.destroy
-
     respond_to do |format|
-      format.html { redirect_to feeds_url, notice: "Feed was successfully destroyed." }
+      if @feed.destroy
```

Comment:
I think it appeared in a different PR ... but noting this "destroy" UI does deviate from the episode.

(The episode status card has a little carat-dropdown thing on the save button, which triggers the delete.  And then you get a bootstrap-modal to confirm rather than the browser built-in).

Should probably open a ticket to convert this one to match?  Or tackle here if you'd rather.

Reply:
added! see changes in `form_status` partial.